dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -1,1878 +1,2434 @@
-apiVersion: v1
-data:
-  rhtap-slo-dashboard.json: |-
+{
```

Comment:
This is not configmap yaml, you copied here just grafana JSON, not configmap

Reply:
the config map is at the end of the file